### DISCUSSION TITLE
March 2022 SP-API Release

### DISCUSSION THREAD

**Discussion by timmyscj** [saleweaver/python-amazon-sp-api#473] [FAQ]
Amazon released new APIs about a week ago. I am interested in learning if the Python Amazon SP-API code will be updated to support the changes. Previous APIs will be deprecated on June 27, 2022.

https://developer-docs.amazon.com/sp-api/changelog/march-2022-sp-api-release-announcement

**Comment by saleweaver** [comment]
Not sure which new APIs were released a week ago, what exactly is it that you mean?

This?:

> 
> This is a reminder that deprecation for the Reports API v2020-09-04 and the Feeds API v2020-09-04 was announced when the 2021-06-30 versions were released in July 2021.
> 
> These two APIs will sunset on July 31, 2022 and calls will fail on August 1, 2022. Please migrate to the [Reports API v2021-06-30](https://developer-docs.amazon.com/sp-api/docs/reports-api-v2021-06-30-reference) and the [Feeds API v2021-06-30](https://developer-docs.amazon.com/sp-api/docs/feeds-api-v2021-06-30-reference) to prevent service
> disruption.

**Reply by timmyscj** [reply]
Yes, that is what I am referring to. For whatever reason the new reportTypes are resulting in a Fatal processingStatus. The original reportTypes work as expected. 

For example, the new GET_VENDOR_SALES_REPORT doesn't work but the original GET_VENDOR_SALES_DIAGNOSTIC_REPORT works fine. I verified using your code as well as Postman. 

I think this is an issue on Amazon's end, not saleweaver's now that I have had more time to troubleshoot.

**Reply by saleweaver** [reply]
Ok, the new endpoint versions are in the library already. If a report is missing, please add it to reportTypes.py and create a PR 🚀. 

Some of the new reports require some strange options, but I haven't found them for all of them yet.

**Reply by timmyscj** [reply]
Will do. Thank you.

**Reply by sktong** [reply]
> Ok, the new endpoint versions are in the library already. If a report is missing, please add it to reportTypes.py and create a PR 🚀.
> 
> Some of the new reports require some strange options, but I haven't found them for all of them yet.

It seems that the GET_FBA_INVENTORY_AGED_DATA and GET_FBA_FULFILLMENT_INVENTORY_HEALTH_DATA currently resulting in a Fatal processingStatus. And the new reportType - GET_FBA_INVENTORY_PLANNING_DATA results in an AttributeError when creating the report.
